lich.rbw

```diff
@@ -100,12 +100,6 @@ class Numeric
 end
 
 class String
-  @@elevated_untaint = proc { |what| what.orig_untaint }
```

Comment:
💡 There's still a lot of 'elevated' variable naming floating around. Not functional, but probably could be renamed in the future to make more sense?

Reply:
I agree. That just makes it a bigger change for now.